scripts/automation/style/pep8.py

```diff
@@ -0,0 +1,29 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def fix_pip8(directory):
```

Comment:
1. I'll fix the typo;
2. I don't think this should be added to the CI script. From what I saw the autopep8 doesn't fix everything perfectly. It needs human intervention.
3. What I do plan to add is a PEP8 code style check run after pylint.